### PR TITLE
Session 1 rails

### PR DESCRIPTION
**Pull request by harishA3**

### CODE REVIEW

File: session_1/hello_world/app/views/page/about_me.html.erb [hunk 1]
```diff
@@ -0,0 +1,2 @@
+<h1>harish</h1>
+<p>B.tech CHEME NITK</p>
```
Comment (raosush) [inline session_1/hello_world/app/views/page/about_me.html.erb:2]: @harishA3 
We need you to add an ordered list below the <p> tag. You can add your interests as an ordered list below the p tag.
You need to add any four of your interests as explained in the reference
Reference - (https://www.w3schools.com/html/html_lists_ordered.asp)

File: session_1/hello_world/app/views/page/root.html.erb [hunk 2]
```diff
@@ -0,0 +1,2 @@
+<h1>welcome to Rails</h1>
```
Comment (raosush) [inline session_1/hello_world/app/views/page/root.html.erb:1]: @harishA3 
Our test expects you to display `Hello, World!`, but you are displaying `Welcome to Rails`
This doesn't matter as you have understood how to display data using HTML tags.

**Comment by raosush** [comment]
@harishA3 
You have successfully completed the task, but our tests expected a set of different results, but that doesn't matter since you have understood on how to display data using HTML tags.

File: session_1/hello_world/app/views/page/about_me.html.erb (hunk 1 above)
Comment (harishA3) [reply session_1/hello_world/app/views/page/about_me.html.erb:2]: I am so sorry. I must have edited it out while editing the html file

Comment (harishA3) [reply session_1/hello_world/app/views/page/about_me.html.erb:2]: Will sure do

**Pull request closed without merging**